vm_test.go

```diff
@@ -531,3 +531,30 @@ func TestStashMemUsage(t *testing.T) {
 		})
 	}
 }
+
+func TestLodashIsPlainObject(t *testing.T) {
```

Comment:
I should probably remove this now, I was troubleshooting why a lodash test was failing in baas.